vscode/src/services/utils/export-logs.ts

```diff
@@ -54,9 +54,25 @@ export async function exportOutputLog(logUri: vscode.Uri): Promise<void> {
             })
     } catch (error) {
         // Open the output channel instead
-        void vscode.commands.executeCommand(
-            'workbench.action.output.show.extension-output-sourcegraph.cody-ai-#1-Cody by Sourcegraph'
-        )
+        openCodyOutputChannel()
         console.error(error)
     }
 }
+
+export function openCodyOutputChannel(): void {
+    void vscode.commands.executeCommand(
+        'workbench.action.output.show.extension-output-sourcegraph.cody-ai-#1-Cody by Sourcegraph'
+    )
+}
```

Comment:
There's a more reliable way to do this that doesn't require hardcoding the extension ID and a magical "#1":

https://github.com/sourcegraph/cody/blob/fcb91b002c9a9da25151bbddf9481ed8b9887230/vscode/src/dev/helpers.ts#L22

```suggestion
export function openCodyOutputChannel(): void {
    outputChannel.show()
}
```